comit/src/network/orderbook.rs

```diff
@@ -251,6 +278,8 @@ pub enum BehaviourOutEvent {
     TakeOrderRequest {
         /// The peer from whom request originated.
         peer_id: PeerId,
+        /// The amount of the order the taker wishes to fill
+        amount: asset::Bitcoin,
```

Comment:
Adding this field with a type of `asset::Bitcoin` makes our `take-order` protocol now specific to BTC/DAI orders.

Unless we take any measures against this, I would propose to change our protocol name to reflect this and make it specific to taking BTC/DAI orders.

Reply:
String would be the most flexible

Reply:
It is not just about the datatype here though but what the format on the wire is and what the other party can "parse" this into.

However, instead of spending time on designing a protocol now that can be reused across several trading pairs, I'd opt to just make it specific now and we can then consolidate later :)